lib/libc/minimal/include/stdlib.h

```diff
@@ -70,6 +70,16 @@ static inline long long llabs(long long __n)
 	return (__n < 0LL) ? -__n : __n;
 }
 
+#if (_POSIX_C_SOURCE - 0) >= 200112
```

Comment:
Just wondering what this magic `(_POSIX_C_SOURCE - 0)` do and if it's possible to drop this zero subtraction?

Reply:
This is a pattern I've seen used in Picolibc and I guess Newlib.

I would be fine using the regular `L` prefixed version as well.

Reply:
@evgeniy-paltsev - fixed!